dev/com.ibm.ws.security.wim.adapter.ldap/src/com/ibm/ws/security/wim/adapter/ldap/LdapConnection.java

```diff
@@ -297,8 +303,9 @@ public NameParser getNameParser() throws WIMException {
      *
      * @param ldapConfigMgr The {@link LdapConfigManager} to get configuration from.
      */
-    public LdapConnection(LdapConfigManager ldapConfigMgr) {
+    public LdapConnection(LdapConfigManager ldapConfigMgr, KerberosService ks) {
```

Comment:
Currently this only sets the KerberosService on initialization. We want it to be able to be updated. So you need to either add a method to allow updating the KerberosService or LdapConnection can take an LdapAdapter instead and get the kerberos service from new helper methods on LdapAdapter.

Reply:
Added new config to receive modify on KerberosService